default-plugins/strider/src/main.rs

```diff
@@ -58,10 +58,14 @@ impl ZellijPlugin for State {
                 Mouse::ScrollUp(_) => {
                     *self.selected_mut() = self.selected().saturating_sub(1);
                 },
-                Mouse::Release(line, _) => {
-                    if line < 0 {
+                Mouse::Release(mut line, _) => {
+                    // we need to shift the line with line -=1 because one line
+                    // is occupied by the current path, and all clicks on it are
+                    // to be ignored
+                    if line < 1 {
                         return;
                     }
+                    line -= 1;
                     let mut should_select = true;
                     if let Some((Event::Mouse(Mouse::Release(prev_line, _)), t)) = prev_event {
                         if prev_line == line
```

Comment:
since line has been decremented, I think prev_line needs to be adjusted too?
I noticed double clicks to enter a directory aren't working reliably.

Reply:
oops, my bad, had the `prev_line` decrement, but accidentally deleted it.

Reply:
@tlinford , done, double clicks seem to be working fine now.